### PR TITLE
Chalcedonian name

### PR DESCRIPTION
**Pull request by IhateTrains**
Renamed Nicene faith to "Chalcedonian", as the game takes place after the Council of Chalcedon in 451 and Oriental Orthodox Churches also belong to Nicene Christianity.

### CODE REVIEW

File: WTWSMS/localisation/WtWSMS_religions.csv [hunk 1]
```diff
@@ -143,10 +143,14 @@ temple_empire_of_sethian;Snake See of;
 d_sethian_holy_order;The Knights Ouroboros;Les chevaliers Ouroboros ;;;;;;;;;x
 d_sethian_holy_order_adj;Snake Knight;Chevalier serpent;;;;;;;;;x
 ###### nicene;;;;;;;;;;;;x
-nicene;Nicene;nic�en;;;;;;;;;;x
-nicene_DESC;Born out of the Council of Nicaea in 325, the Nicene Christianity is the central church of the Pentarchy, and the main faith of the two empires.;N�e lors du concile de Nic�e en 325, le christianisme nic�en est l'�glise centrale de la Pentarchie, et la foi principale des deux empires.;;;;;;;;;;x
+#nicene;Nicene;nic�en;;;;;;;;;;x
```
Comment (IhateTrains) [inline WTWSMS/localisation/WtWSMS_religions.csv]: Damn, apparently broke encoding. Will fix.

Comment (IhateTrains) [reply WTWSMS/localisation/WtWSMS_religions.csv]: I still don't see why some "é"s appear as "é" and some as "�", even in the unchanged lines (for example: d_orthodox;Ecumenical Patriarchy;Patriarcat �cum�nique;;;;;;;;;;x
d_orthodox_adj;Ecumenical;Œcuménique;;;;;;;;;;x)
@loup99 any idea

Comment (loup99) [reply WTWSMS/localisation/WtWSMS_religions.csv]: No idea.

Comment (IhateTrains) [reply WTWSMS/localisation/WtWSMS_religions.csv]: I tried redoing the changes in another branch but the result was the same.
I run the game with French language, however, and the names are fine in the game:
image

File: WTWSMS/localisation/WtWSMS_religions.csv [hunk 2]
```diff
@@ -148,7 +148,7 @@ d_sethian_holy_order_adj;Snake Knight;Chevalier serpent;;;;;;;;;x
 #k_nicene;The Nicene Church;L'�glise Nic�en;;;;;;;;;;x
 #k_nicene_adj;Nicene;Nic�en;;;;;;;;;;x
 nicene;Chalcedonian;chalc�donienne;;;;;;;;;;x
-nicene_desc;Born out of the Council of Chalcedon in 451, the Chalcedonian Christianity is the central church of the Pentarchy, and the main faith of the two empires.;;;;;;;;;;;x
+nicene_DESC;Born out of the Council of Chalcedon in 451, the Chalcedonian Christianity is the central church of the Pentarchy, and the main faith of the two empires.;N�e lors du concile de Chalc�doine en 451, le christianisme chalc�donienne est l'�glise centrale de la Pentarchie, et la foi principale des deux empires.;;;;;;;;;;x
```
Comment (loup99) [inline WTWSMS/localisation/WtWSMS_religions.csv]: It should be _chalcédonien_ in masculine, not _chalcédonienne_ in feminine.

Comment (IhateTrains) [reply WTWSMS/localisation/WtWSMS_religions.csv]: Still simpler than in my language...
Fixed ✔️

File: WTWSMS/localisation/WtWSMS_religions.csv (hunk 1 above)
Comment (loup99) [reply WTWSMS/localisation/WtWSMS_religions.csv]: Yes, that works exactly as it should (except for the second grammar mistake I spotted 😛 ).

File: WTWSMS/localisation/WtWSMS_religions.csv (hunk 2 above)
Comment (loup99) [reply WTWSMS/localisation/WtWSMS_religions.csv]: Thanks, seems like there is an older mistake too, _Née_ should be _Né_.

Comment (IhateTrains) [reply WTWSMS/localisation/WtWSMS_religions.csv]: Done, also fixed in the previously used line.

Comment (loup99) [reply WTWSMS/localisation/WtWSMS_religions.csv]: I don't want to nitpick with French grammar, but while you are at it nicene should also be _chalcédonien_ as well as k_nicene_adj. (k_nicene should remain _L'Église chalcédonienne_) That will be the last comment I make, I promise. 🙄

Comment (IhateTrains) [reply WTWSMS/localisation/WtWSMS_religions.csv]: Should be corrected now ✔️ 
TIL there are no grammatical cases in French
image
I don't get why Hungarians would want 18.

Comment (loup99) [reply WTWSMS/localisation/WtWSMS_religions.csv]: We are fortunate in that regard, yes. 😄 That being said silent consonants and such complicate the spelling enough for French to be a challenge anyway. :wink:

**Pull request merged**